src/__tests__/index.spec.js

```diff
@@ -210,16 +211,17 @@ function setupGetMountedComponent(TYPE, mockListeners = mockListenersSetup) {
     it('calls preventDefault when onSwiping is present', () => {
       const onSwiping = jest.fn()
       const preventDefault = jest.fn()
+      const cancelable = jest.fn()
```

Comment:
💡 mind updating this and the other test usage to `boolean`'s? `const cancelable = true`?

https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelable

Reply:
Sure, my bad. I'll change it now and update the PR.

Reply:
PR updated!